library/src/lib/time/time.component.html

```diff
@@ -107,13 +107,13 @@
                 [attr.aria-label]="timeI18nLabels?.increasePeriodLabel"></button>
     </div>
     <div class="fd-time__input">
-        <input [(ngModel)]="period"
+        <input [ngModel]="period"
```

Comment:
Why did you remove two-way binding here?

Reply:
Usually using ngModelChange on top of two-way binding doesn't hurt

Reply:
I don't want to override period variable, when it's invalid.